editor/js/editor-libs/events.js

```diff
@@ -24,6 +24,14 @@ function addCSSEditorEventListeners(exampleChoiceList) {
 
     exampleChoiceList.addEventListener('click', function(event) {
         var target = event.target;
+        var isClickedBetween = target.getAttribute('id') === 'example-choice-list';
+
+        // if original target is in between or outside of `example-choice` 
+        // elements, ie the parent <section id='example-choice-list'> element
+        // then bail
+        if (isClickedBetween) {
+            return;
+        }
 
         // if the original target is not an `example-choice` element
         if (!target.classList.contains('example-choice')) {
```

Comment:
@tannerdolby What I find curious is how we even get into this conditional if the `target` does not contains a class with the name `example-choice`

Reply:
I found that a bit curious as well. It works, but is confusing at first glance. Its essentially saying if a user clicks inside the `<code>` element in example choices and not within the parent `<div class="example-choice">` container then find the nearest parent div with `findParentChoiceElem`. I will include a demo below with some console logs to highlight this.

Reply:
@tannerdolby What if we do this. We add the following function:

```
function handleChoiceEvent() {
    if (this.classList.contains('copy')) {
        mceAnalytics.trackEvent({
            category: 'Interactive Example - CSS',
            action: 'Copy to clipboard clicked',
            label: 'Interaction Events',
        });
    }

    // and pass it on to `onChoose`
    module.exports.onChoose(this);
}
```

and then we replace this block of code with:

```
const exmapleChoices = exampleChoiceList.querySelectorAll(
        '.example-choice'
    );
    Array.from(exmapleChoices).forEach((choice) => {
        choice.addEventListener('click', handleChoiceEvent);
    });
```

Reply:
Instead of `Array.from(exampleChoices)` could we just convert the static NodeList to an array with brackets and the spread operator like `[...exampleChoiceList.querySelectorAll('.example-choice')]`? Either way is fine just wanted to see what your thoughts were.

Reply:
And it wasn't entirely clear, when you say replace 'X' with 'Y', are you referring to replacing 

```js
 var isClickedBetween = target.getAttribute('id') === 'example-choice-list';

        // if original target is in between or outside of `example-choice` 
        // elements, ie the parent <section id='example-choice-list'> element
        // then bail
        if (isClickedBetween) {
            return;
        }
```

with 

```js
const exampleChoiceList = [...document.querySelectorAll('.example-choice')];
        exampleChoiceList.forEach((choice) => {
            choice.addEventListener("click", handleChoiceEvent);
        });
```

Reply:
Since the spread operator will expand the array literal `...stuff` and then using `[]` we create a new array by placing the "spread" values inside brackets like `[...stuff]` this will create a **full** copy on every element. Because `Array.from(arg)` creates a new array from its argument as a "shallow" copy, I reckon your original choice of using `Array.from()` will be best. Sorry for the rant!